registration/forms.py

```diff
@@ -43,7 +43,7 @@ class BetaTestApplicationForm(NgModelFormMixin, NgFormValidationMixin, NgModelFo
     """
     class Meta:
         model = BetaTestApplication
-        exclude = ('user', 'status')
+        exclude = ('user', 'status', 'instance')
```

Comment:
This was a change silently dropped by git during the rebase,  It already existed before rebasing.  Without this change, the form still kind of works, but behaves slightly weird, and some of the messages shown at the top of the form disappear.  I've got no idea why this happens, and it was a pain to debug, since I simply didn't expect git to silently drop changes (and I still don't no how this can have happened).